src/legacy/server/saved_objects/saved_objects_mixin.js

```diff
@@ -84,8 +78,13 @@ export function savedObjectsMixin(kbnServer, server) {
 
   const provider = kbnServer.newPlatform.__internals.savedObjectsClientProvider;
 
+  const importAndExportableTypes = typeRegistry
```

Comment:
Yea, this is a temporary measure for BWC until the client side of the plugin is migrated. This will be removed in favor of an http call to retrieve the types (implemented in the https://github.com/elastic/kibana/pull/59110 sandbox)